SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1153,12 +1154,13 @@ void USpatialReceiver::OnComponentUpdate(const Worker_ComponentUpdateOp& Op)
 	case SpatialConstants::AUTHORITY_INTENT_COMPONENT_ID:
 		if (NetDriver->IsServer())
 		{
-			check(LoadBalanceEnforcer);
-			LoadBalanceEnforcer->OnAuthorityIntentComponentUpdated(Op);
+			TSharedPtr<SpatialLoadBalanceEnforcer>LBEnforcer = LoadBalanceEnforcer.Pin();
+			check(LBEnforcer.IsValid());
+			LBEnforcer->OnAuthorityIntentComponentUpdated(Op);
 		}
 		return;
 	case SpatialConstants::VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID:
-		if (NetDriver->VirtualWorkerTranslator != nullptr)
+		if (NetDriver->VirtualWorkerTranslator.IsValid())
```

Comment:
Perhaps this is still to come but I guess SpatialReciever wants a weak pointer to VirtualWorkerTranslator on it as a member.

Reply:
Yes, it should be a weak pointer to the Translator and it's still to come 👍

Reply:
Added to summary below.